components/ingest-service/integration_test/project_update_test.go

```diff
@@ -1489,7 +1489,7 @@ func TestProjectUpdatePainlessElasticsearchScript(t *testing.T) {
 					jobStatus, err = suite.ingest.JobStatus(ctx, esJobID)
 					assert.Nil(t, err)
 					if err != nil {
-						assert.FailNow(t, "testing job complete")
```

Comment:
This is the only real change in this file. The rest of the changes are from golang's code simplifier.